scripts/bash/gitlab_backup_exec.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
Why is this a different file and not combined with gitlab_backup?

Reply:
So it will be more convenient to modify the backup bash script through Terraform instead of accessing the instance or redeploy it.

The gitlab_backup_exec.sh is placed on the instance and referenced from a cron job that set during the deployment.
It downloads the gitlab_backup.sh file on every backup process.